docs/localization-translation/README.md

```diff
@@ -0,0 +1,181 @@
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/2016-02-17-at-09.59.png)
+
+4. Scroll to the bottom for export options. Export a `.mo` file for use on your site.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/2016-02-17-at-10.00.png)
+
+5. Rename this file to `woocommerce-YOURLANG.mo` (e.g., Great Britain English should be `en_GB`).
+6. Upload to your site under `wp-content/languages/woocommerce/`. Once uploaded, this translation file may be used.
+
+## Creating custom translations
+
+WooCommerce includes a language file (`.pot` file) that contains all of the English text. You can find this language file inside the plugin folder in `woocommerce/i18n/languages/`.
+
+### Creating custom translations with Loco Translate
+
+1. Install and activate the [Loco Translate](https://wordpress.org/plugins/loco-translate/) plugin.
+2. Go to the new Loco Translate menu item created in your Dashboard with five sub-sections.
+3. Go to the `Plugins` section, and select `WooCommerce`.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/loco-translate-woocommerce.png)
+
+4. Add new languages using the `Add New Language` link.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/loco-translate-new-language.png)
+
+5. Choose the language to add in the list, or enter the language ISO code (ex: `fr_FR`, `en_US` etc.), select the folder to add the translation files to, and click `Start Translating`.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/loco-translate-create-language.png)
+
+    > **Note:**
+    >
+    > - Loco Translate offers three different locations for where to create the translation. The best choice for your own translation is Custom, not just to prevent custom translations from being undone by updates, but also because it’s needed for strings in extensions like WooCommerce Subscriptions to be properly translated. Avoid the Author location since it’s inside the plugin and will be overwritten by extension updates, as well as the System location, which could be overwritten by translations from translate.wordpress.org.
+    > - If you encounter any technical issues while using Loco Translate, please get in touch with [the Loco Translate support team](https://wordpress.org/support/plugin/loco-translate/).
+
+6. Save when finished.
+
+    ![screenshot](https://woocommerce.com/wp-content/uploads/2012/01/loco-translate-translating-1.png)
+
+    > **Note:** When a new version of WooCommerce is released and updated on your site, you need refresh the .po file to look for new strings. Use the “Sync” button to find any new string to start translating.
+
+## Creating custom translations with PoEdit
+
+[Poedit ](https://poedit.net/) is a more advanced alternative to the [Loco Translate](https://wordpress.org/plugins/loco-translate/) plugin. Only advanced users or translators wishing to create a custom translation to WooCommerce should attempt this method.
```

Comment:
> Only advanced users or translators wishing to create a custom translation to WooCommerce should attempt this method.

I would drop this superlative sentence for a couple reasons:

- Poedit has already been referred to as an advanced alternative.
- Loco Translate has already been referred to as a way to create a custom translation (using Poedit is not distinctive on that point).

Reply:
I've removed the superlative sentence.